src/program/snabbvmx/lwaftr/lwaftr.lua

```diff
@@ -99,6 +109,9 @@ function run(args)
    if file_exists(conf_file) then
       conf = lib.load_conf(conf_file)
       if not file_exists(conf.lwaftr) then
+         conf.lwaftr = dirname(conf_file).."/"..conf.lwaftr
```

Comment:
A comment would be useful to clarify that if the file does not exist in the current directory, it will be looked up in the main config file directory.